packages/shopify-codemod/test/fixtures/arguments-to-args-spread/basic.input.js

```diff
@@ -0,0 +1,59 @@
+function shouldGetArgsSpread() {
+  foo(arguments);
+}
+
+function shouldGetArgsSpread() {
+  foo(...arguments);
+}
+
+function shouldGetArgsSpread() {
+  console.log(arguments);
+  function arg1ShouldPreventRenamesInsideThisScope(arg1) {
+    console.log(arguments);
+  }
+}
+
+function foo() {
+  (function() { // Function IIFE has its own scope.
+    console.log(arguments);
+  })();
+}
+
+function foo() {
+  console.log(arguments);
+  (() => { // Arrow IIFEs with no arguments inherit parent's arguments.
+    console.log(arguments);
+    (() => {
+      console.log(arguments);
+      (() => {
+        console.log(arguments);
+      })();
+    })();
+  })();
+}
+
+function arrowIIFEParentWithArgsSpread(...args) {
+  (() => { // Arrow IIFE with no arguments inherits parent scope's arguments.
+    console.log(arguments);
+  })();
+}
+
+function foo() {
+  (() => {
+    (() => {
+      console.log(arguments);
+    })("arg1"); // Arrow IIFE with arguments does not inherit parent's arguments.
```

Comment:
Maybe I am misreading this comment, but doesn't an arrow function always have its parent version of arguments? This doesn't seem to make sense given that you are (correctly) transforming this to use `args`, unless I am misreading.